pytest_reportportal/plugin.py

```diff
@@ -49,12 +49,7 @@ def get_launch_attributes(rp_launch_attributes):
     """
     launch_attrs = []
     for rp_attr in rp_launch_attributes:
-        try:
-            key, value = rp_attr.split(':')
-            attr_dict = {'key': key, 'value': value}
-        except ValueError:
-            attr_dict = {'value': rp_attr}
-
+        attr_dict = PyTestServiceClass._get_attr_dict(rp_attr)
```

Comment:
Why do not use re-use that function instead of modification?

Reply:
What do you mean? You want me to call get_launch_attributes in _get_marker_value? I don't think that is a good idea. First, I'm not sure if I can call the whole get_launch_attributes function. Second, there is a problem of accessing plugin from service module. In fact right now there is no such functions in plugin that are being called from service. On the other hand whole PyTestServiceClass is being imported in plugin module.

Reply:
What I mean is that importing get_launch_attributes into service module causes cross import error